### ISSUE TITLE
Realism: Pine Lean-To

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#53937]
### Describe the bug

The pine lean-to does not seem realistic. Pines, especially taller ones, tend to have their "initial" branches up high so using them as a shelter with just a few sticks needs to be nerfed or removed. There is no way it could shelter one from rain.

### Steps to reproduce

n/a

### Expected behavior

Spruce trees (It appears they actually do grow in New England) seem to be a bit better suited for this, but this is still far from optimal in terms of cover from rain. I did some research and could not find anything related to this, except some shelters similar to the already existing makeshift shelter.

Another relatively minor issue this construction causes is that you can transform pines into walkable terrain. The recipe should place the shelter adjacent to the pine instead.

I think it might be better to just remove this recipe, which originated from #30951. Opinions?

### Versions and configuration

n/a

**Comment by kevingranade** [comment]
In broad strokes there is a place for a lean-to for very low level wilderness survival and zero tools, but this recipe does have some issues.

The number of branches is too small, and they need to be long branches.  Ideally an even larger sapling log or very large branch item, but we don't have that.
It shouldn't care about the kind of tree.
Ideally these would be notably suboptimal compared to even the improvised shelter.

**Comment by NetSysFire** [comment]
I agree but I think it might be better to just remove it because even without tools it is trivial to get the requirements for the improvised shelter. Gathering as many long sticks could take longer than levelling survival and fabrication so you can construct the improvised shelter. The result is *significantly* inferior than everything else and there is no real reason for the player to use this.

**Comment by Coseo** [comment]
Having spent a lot of time in flagstaff arizona with relatives that live in Coconino county, in the middle of the world's largest contiguous Ponderosa pine forest, I can say that I've spent a lot of time around (that particular subtype of) pine trees. While  _most_ of the large branches are indeed up high, you'll usually be surrounded by at least a few trees with several-foot-long branches near ground level, and if you have access to a lot to pick from, you could easily get dozens of 4+ feet long branches. That's been my experience spending hours as a kid wandering around that forest - and as a little kid, taking entire walks in which I did nothing but break large low branches off of the trees with a walking stick for fun. In a single hour long walk I'd probably break a few hundred. 

I can't speak to the exact subtypes seen in New England, but having done some searching online, the images I've seen suggest that at least _some_ of those are capable of having low hanging branches without even seeking out still-growing-trees, which you could substitute entirely or using younger ones which will absolutely have low branches. Failing all of that, climbing the trees or chopping them down would make this trivial - at that point you could make a shelter a lot of ways, though. 

Anyways, unless someone who lives in new england and walked their red pine forests can confirm that in fact those particular types of pines _never_ behave this way, I'd like to really ask this be looked into further, because if they're anything like the pine forests I visit every year in arizona, this suggestion would be really off the mark.

**Comment by NetSysFire** [comment]
Damn you stale bot. I will probably submit a PR to remove the pine lean-to very soon. There will be no replacement because of the reasons above, this "collect 50 sticks for a bad shelter" just does not make any sense. In terms of migration.json replacement I think I will choose the tarp lean-to if that is possible at all with terrain.

**Comment by NetSysFire** [comment]
These are JSON properties since #53612 was merged, so you will find the values in their terrain or furniture definition.

**Comment by evankimori** [comment]
> These are JSON properties since #53612 was merged, so you will find the values in their terrain or furniture definition.

Magnificent. Thank you for your quick response and patience with a newbie. :)
